src/main/java/edu/greenblitz/tobyDetermined/IsRobotReady.java

```diff
@@ -3,32 +3,32 @@
 
 import edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid.Grid;
 import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
-import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Elbow;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.ElbowSub;
```

Comment:
Raz - Why keep the sub in the name of the subsystem?

Reply:
changed to elbow